File: Java/1556.java
class Solution {
    public String thousandSeparator(int n) {
        String word = "",m = String.valueOf(n);
        int l = m.length();
        for(int i = 0; i < l; i++){
            word += m.charAt(i);
            if((l-i-1)%3 == 0 && i+1 < l) word += ".";
        }
        return word;
    }
}

File: Python/1556.py
class Solution(object):
    def thousandSeparator(self, n):
        word = ""
        l = len(str(n))
        for i in range(l):
            word += str(n)[i]
            if (l-i-1)%3 == 0 and i+1<l:
                word += "."
        return word